docs/source/howto/level1-csd.rst

```diff
@@ -0,0 +1,312 @@
+
+There are two further optional arguments that don't affect the search parameters, but change the behavior of FORCE Level 1 CSD:
+
+.. code-block:: none
+
+  -n | --no-act
+  Will trigger a dry run that will only return the number of images
+  and their total data volume
+
+  -k | --keep-meta
+  Will write the results of the query to the level 1 datapool directory.
+  Two files will be created if Landsat and Sentinel-2 data is queried
+  at the same time. Filename: csd_metadata_YYYY-MM-DDTHH-MM-SS
+
+
+**Please note** that the mandatory arguments are positional! They need to be provided in this exact order. The optional arguments can be placed anywhere and may also be combined. For example, ``-n -k -c 0,70`` could also be written as ``-nkc 0,70``. When values are passed to the optional arguments (cloud cover, date range, sensor, or tier), these must be separated by commas ``,`` and must not contain whitespace.
+
+Querying and downloading data
+-----------------------------
+
+Basic search
+""""""""""""
+
+Now that the FORCE Level 1 CSD is set up and we got the basic syntax straight, we can start looking for scenes that match our needs. Let's start with a very broad search using a geopackage file defining our area of interest (the city-state of Berlin, Germany) as polygon feature. You may also use point or line features as aois.
+
+To get a first overview of the data volume, we will use the ``-n | --no-act`` option. This will print information about the results of our search without actually downloading the data.
+
+
+.. code-block:: none
+
+    force-level1-csd meta gcs_berlin/level1 gcs_berlin/level1/l1_pool.txt gcs_berlin/berlin_aoi.gpkg -n
+
+    Landsat and Sentinel-2 data requested.
+    Landsat data will be queried and downloaded first.
+
+    Searching for footprints / tiles intersecting with geometries of AOI shapefile...
+
+    Querying the metadata catalogue for Landsat data
+    Sensor(s): LT04,LT05,LE07,LC08
+    Tier(s): T1
+    Tile(s): 192023,192024,193023,193024
+    Daterange: 19700101 to 20201013
+    Cloud cover minimum: 0%, maximum: 100%
+
+    2510 Landsat Level 1 scenes matching criteria found
+    827.38GB data volume found.
+
+    Searching for footprints / tiles intersecting with geometries of AOI shapefile...
+
+    Querying the metadata catalogue for Sentinel2 data
+    Sensor(s): S2A,S2B
+    Tile(s): T32UQD,T33UUT,T33UUU,T33UVT,T33UVU
+    Daterange: 19700101 to 20201013
+    Cloud cover minimum: 0%, maximum: 100%
+
+    4152 Sentinel2 Level 1 scenes matching criteria found
+    1.78TB data volume found.
+
+    Done.
+
+
+FORCE Level 1 CSD will print all details about the query parameters, as well as number of scenes found and total data volume for Landsat and Sentinel-2 data.
```

Comment:
Now that I am reading this. You actually can implement another fail-safe that was generally not doable with the other FORCE programs, i.e. check if we have enough space to actually store the downloaded data :) Can be added at a later point, though.

Have a look at ``man df``

Reply:
I knew I should've left this detail out :p